### PR TITLE
brought back $barOptions

### PR DESCRIPTION
**Pull request by simialbi**
updated modal code style (removed double quotes)
corrected author meta info

| Q             | A
| ------------- | ---
| Is bugfix?    | ❌ 
| New feature?  | ✔️ 
| Breaks BC?    | ❌ 
| Tests pass?   | ✔️  
| Fixed issues  | #95

### CODE REVIEW

**Comment by samdark** [comment]
Tests actually passed...

File: src/Progress.php [hunk 1]
```diff
@@ -25,22 +25,20 @@
  *
  * // styled
  * echo Progress::widget([
- *     'bars' => [
```
Comment (samdark) [inline src/Progress.php]: I'd leave documentation for using 'bars' as well.

File: src/Progress.php [hunk 2]
```diff
@@ -67,6 +65,11 @@ class Progress extends Widget
      * @var int the amount of progress as a percentage.
      */
     public $percent = 0;
+    /**
+     * @var array the HTML attributes of the bar.
+     * @see \yii\helpers\Html::renderTagAttributes() for details on how attributes are being rendered.
```
Comment (samdark) [inline src/Progress.php]: Need "since" tag.

Comment (samdark) [reply src/Progress.php]: Also I'd note that it is used in case `bars` is empty only.

**Comment by simialbi** [comment]
> Tests actually passed...

Ok in my commit in mine repository tests did not pass:
https://travis-ci.com/simialbi/yii2-bootstrap4/builds/119444487

File: docs/guide/migrating-yii2-bootstrap.md [hunk 3]
```diff
@@ -18,3 +18,9 @@ The most notable changes are summarized below:
 * [[yii\bootstrap4\Breadcrumbs|Breadcrumbs]] was added (Bootstrap 4 implementation of [[yii\widgets\Breadcrumbs]])
 * [[yii\bootstrap4\ButtonToolbar|ButtonToolbar]] was added (https://getbootstrap.com/docs/4.2/components/button-group/#button-toolbar)
 
+## Changes in widgets
+
+### Modal
+
+The Modal has no `$header` property any more. It was renamed to `$title`. It isn't necessary to write the `<h2 class="modal-title>`
```
Comment (samdark) [inline docs/guide/migrating-yii2-bootstrap.md]: That should be considered backwards compatibility break.

File: src/Progress.php [hunk 4]
```diff
@@ -55,18 +77,24 @@
...
     public $label;
     /**
-     * @var int the amount of progress as a percentage.
+     * @var int the amount of progress as a percentage. This property will only be considered if [[bars]] is empty
      */
     public $percent = 0;
+    /**
+     * @var array the HTML attributes of the bar. This property will only be considered if [[bars]] is empty
+     * @see \yii\helpers\Html::renderTagAttributes() for details on how attributes are being rendered.
+     * @since 2.0.6
```
Comment (samdark) [inline src/Progress.php]: Can't be 2.0.6 because of property removal. 2.1.0.

File: docs/guide/migrating-yii2-bootstrap.md (hunk 3 above)
Comment (simialbi) [reply docs/guide/migrating-yii2-bootstrap.md]: It's just docs. THe backwards compatibility break is since the first release

Comment (samdark) [reply docs/guide/migrating-yii2-bootstrap.md]: Ah, fine then :)

**Comment by samdark** [comment]
Merged. Thank you!

**Pull request merged**